docs/ide-usage.md

```diff
@@ -78,6 +78,7 @@ The following plugins are needed:
    ![setupguide-6.png](images/setupguide-6.png)
 
 1. To set up some static analysis tools, refer to [this document](static-analysis.md).
+1. To move on to development phase, refer to [this document](development.md).
```

Comment:
... move on to **the** development phase ...
Also, add a blank line before this line.

Reply:
Sir @whipermr5, I think that adding a blank line between point 5 and point 6 will not be of any use.

Reply:
If it is not part of point 5, it should be separated from point 5 by a blank line for consistency with the previous parts of the document.